sql/core/src/test/resources/sql-tests/inputs/cte-command.sql

```diff
@@ -0,0 +1,33 @@
+-- WITH inside CTE
```

Comment:
QQ: Is there a case will fail before the code change?

Reply:
no, but the analyzed plan is different, as we always inline CTE before.